scripts/generate-framework-agreement-signature-pages.py

```diff
@@ -36,13 +36,16 @@
 If supplied, <supplier_id_file> is a text file with one supplier ID per line; framework agreement pages will only be
 generated for these suppliers.
 
+Alternatively you can supply one or more supplier IDs at the command line using the `--supplier-id` flag.
+
 Usage:
     scripts/generate-framework-agreement-signature-pages.py <stage> <framework_slug> <output_folder>
-    <path_to_agreements_repo> [<supplier_id_file>]
+    <path_to_agreements_repo> [(--supplier-id=<id>... | <supplier_id_file>)]
```

Comment:
I can't get this to run locally with the ID or the file 🤔 I might be doing something wrong?

`./scripts/generate-framework-agreement-signature-pages.py local digital-outcomes-and-specialists-3 output ../digitalmarketplace-agreements --supplier-id=710702`